src/aces/query.py

```diff
@@ -137,6 +139,23 @@ def query(cfg: TaskExtractorConfig, predicates_df: pl.DataFrame) -> pl.DataFrame
         )
         to_return_cols.insert(1, "label")
 
+        if result["label"].n_unique() == 1:
```

Comment:
Just add a `# pragma: no cover` comment at the end of this line and related lines and it will ignore blocks like this that are just logging. https://coverage.readthedocs.io/en/latest/excluding.html

Reply:
Thanks, this is helpful!